source/_patterns/01-molecules/tag-list/tag-list.twig

```diff
@@ -1,4 +1,4 @@
-{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['tag-list']) }) -%}
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['item-tags']) }) -%}
```

Comment:
This is confusing, seems like two patterns are being mixed up.

Reply:
Reverted back to previous class